docs/user-guide/docker-volume-cli.md

```diff
@@ -71,3 +71,11 @@ docker volume create --driver=vmdk --name=MyVolume -o size=10gb -o fstype=ext4 (
 
 Specifies which filesystem will be created on the new volume. vSphere Docker Volume Service will search for a existing /sbin/mkfs.**fstype** on the docker host to create the filesystem, and if not found it will return a list of filesystems for which it has found a corresponding mkfs. The specified filesystem must be supported by the running kernel and support labels (-L flag for mkfs). Defaults to ext4 if not specified. 
 
+### clone-from
+```
+docker volume create --driver=vmdk --name=CloneVolume -o clone-from=MyVolume -o access=read-only
```

Comment:
Not sure if you have multi-datstore setup. Did you test following;

`docker volume create --driver=vmdk --name=CloneVolume@DS2 -o clone-from=MyVolume@DS3`